packages/jss-plugin-rule-value-function/src/index.js

```diff
@@ -55,6 +56,13 @@ export default function functionPlugin() {
         // Empty object will remove all currently defined props
         // in case function rule returns a falsy value.
         styleRule.style = fnRule(data) || {}
+
+        for (const prop in styleRule.style) {
```

Comment:
We could avoid doing these checks in production by wrapping it in  `if (process.env.NODE_ENV === 'development')`

Reply:
Then we could even avoid warning package and use only `console.warn`.

Reply:
I would keep warning for consistency, but it doesn't matter technically, just in case in the future we want to replace warning, it will be easier if it's used consistently